site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,237 @@
+  </nav>
+  <nav class="navbar navbar-dark bg-dark navbar-expand-lg" aria-label="Global navigation">
+    <div class="container-xxl">
+      <div class="navbar-brand me-auto me-lg-4">
+        <a class="stretched-link" href="#">
+          <img src="/docs/{{< param docs_version >}}/assets/brand/orange-logo.svg" width="50" height="50" alt="Boosted - Back to Home"
+            loading="lazy">
+        </a>
+        <h1 class="title">Travel</h1>
+      </div>
+      <button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse" data-bs-target=".global-header-1"
+        aria-controls="global-header-1.1 global-header-1.2" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div id="global-header-1.1" class="navbar-collapse collapse me-lg-auto global-header-1">
+        <ul class="navbar-nav">
+          <li class="nav-item"><a class="nav-link" href="#">All inclusive cards</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Data only cards</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Top up</a></li>
+          <li class="nav-item"><a class="nav-link" href="#">Assistance</a></li>
+        </ul>
+        <ul class="navbar-nav border-top border-1 border-dark d-flex flex-row d-lg-none">
+          <li class="nav-item"><a class="nav-link active" href="#" aria-label="EN English version" aria-current="true">EN</a></li>
+          <li class="nav-item"><a class="nav-link" href="#" aria-label="FR Version française" lang="fr" hreflang="fr">FR</a></li>
+        </ul>
+      </div>
+      <div id="global-header-1.2" class="navbar-collapse collapse d-sm-flex global-header-1">
+        <ul class="navbar-nav flex-row">
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#buy"></use>
+              </svg>
+              <span class="visually-hidden">Basket</span>
+            </a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link nav-icon">
+              <svg width="1.5rem" height="1.5rem" fill="currentColor" aria-hidden="true" focusable="false"
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#user"></use>
+              </svg>
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+<main class="bd-content order-1">
+  <div class="bg-white title-bar">
+    <div class="container-xxl d-block">
+      <div class="row">
+        <div class="col-md-5 offset-md-3">
+          <h1 class="display-1">Become a reseller</h1>
```

Comment:
Seems to be `display-2` 🤷 But I think this wasn't specified in DSM. The design should be 60px or 50px in a viewport of 1024px not 1280px ...

Reply:
We used `display-1` in pages already online with similar design: [https://boosted.orange.com/docs/5.2/examples/](https://boosted.orange.com/docs/5.2/examples/) or [https://boosted.orange.com/docs/5.2/guidelines/](https://boosted.orange.com/docs/5.2/guidelines/)

If we decide to change it we will have to change it everywhere so it may be wiser to do it later in a distinct pull request?

Reply:
Agreed. Let's do this when we'll tackle the overall typography work with the designers. Resolving it.